test/fields/mutation.test.ts

```diff
@@ -439,6 +439,32 @@ describe('field mutation', () => {
         hint: 'Optional contact method',
       })
     })
+
+    it('should be able to preserve x-jsf-presentation properties in a nested field when mutating the schema', () => {
```

Comment:
```suggestion
    it('should be able to preserve x-jsf-presentation properties in a nested field', () => {
```

No need to say it's mutating the schema, this is just a schema ;)